src/exchange/offerMake/offerMake.test.ts

```diff
@@ -1152,7 +1152,7 @@ test('place direct sell order that exceeds the balance', () => {
 
   expect(unpack(controller).amount).toEqual(new BigNumber(12));
   expect(unpack(controller).price).toEqual(new BigNumber('0.18333333333333333333'));
-  expect(unpack(controller).total).toEqual(new BigNumber('2.19999999999999999996'));
+  expect(unpack(controller).total).toEqual(new BigNumber('2.2'));
```

Comment:
I'm very curious why this was changed

Reply:
Because slippageLimit was increased by 0.001%